pallets/settlement/src/benchmarking.rs

```diff
@@ -589,167 +566,6 @@ benchmarks! {
             [parameters.portfolios.sdr_portfolios.clone(), parameters.portfolios.sdr_receipt_portfolios].concat();
     }: _(alice.origin, InstructionId(1), parameters.portfolios.sdr_portfolios[0])
 
-    ensure_allowed_venue {
```

Comment:
This benchmarks were temporary (created in the WeightMeter PR) and are not being used.